### ISSUE TITLE
php artisan infyom.publish:tests (Laravel 5.4)

### ISSUE THREAD

**Issue by DanielBoettner** [InfyOmLabs/laravel-generator#463]
As documented in http://labs.infyom.com/laravelgenerator/docs/master/generator-options

will cause:

```
[Symfony\Component\Console\Exception\CommandNotFoundException]
  Command "infyom.publish:test" is not defined.
  Did you mean one of these?
      infyom.api:tests
      infyom.publish:layout
      infyom.publish:templates
      infyom.publish:vuejs
      infyom:publish
      make:test
```

While  trying 'infyom.api:test' will cause:

```
[Symfony\Component\Console\Exception\RuntimeException]
  Not enough arguments (missing: "model").
```

**Comment by alvaromartinezmajado** [comment]
I believe `infyom.api:test`is throwing that exception because it's meant for generating the test files so a Model has to be provided. However I can reproduce the same issue with `php artisan infyom.publish:tests` and tests seem to be broken in 5.4: even when `ApiTestTrait` is present and `autoload-dev` set in the `composer.json` as indicated at http://labs.infyom.com/laravelgenerator/docs/5.4/generator-options, I'm getting `Trait 'ApiTestTrait' not found` errors.

**Comment by skys215** [comment]
I have the same problem while upgrading to Laravel 5.5.

**Comment by skys215** [comment]
OK, now I can run the test.
I changed the base class of `TestCase` which `laravel/browser-kit-testing/src/TestCase.php` was using. And I found that this change has been made in `master` branch of [this file](https://github.com/laravel/browser-kit-testing/commit/935dff36bf64d2bbbc04cc3b5370bd70f13bdb4c#diff-d88dadea11088dc7a4f6d1070956b2bf). What's more, I found it was done after the 1.3 version of `browser-kit-testing` package. 
So I let composer use the latest version of `browser-kit-testing` and the tests can run.
